packages/tsx/tsx.mjs

```diff
@@ -36,19 +36,11 @@ async function resolveTSX(specifier, ctx, nextResolve) {
 		/** @type {FileURL} */ (ctx.parentURL ?? pathToFileURL(path.join(cwd(), 'whatever.ext')).href),
 	);
 
-	if (jsxExts.has(ext)) {
+	if (ext === '.jsx' || ext === '.tsx' || ext === '.mts' || ext === '.ts') {
```

Comment:
why switch from a Set to a bunch of inline ORs?

Reply:
The OR list is a little bit faster (but nothing at all). It's mainly for reading purposes, because here you can see straight away which extensions are supported. If you keep as before, you have to see the definition of the variable before you can understand what it does